pipeline/splicing/calcVarPriors.py

```diff
@@ -29,11 +38,69 @@ def getVarStrand(variant):
     if varGene == "BRCA1": 
         return '-'
     elif varGene == "BRCA2":
-        return '+'
+         return '+'
     else:
         return ""
 
 
+def _make_request(url):
+    '''Makes request to API and returns json file'''
+    req = requests.get(url, headers = {"Content-Type": "application/json"})
+    
+    if req.status_code == 429 and 'Retry-After' in req.headers:
+        retry = float(req.headers['Retry-After'])
+        time.sleep(retry)
+        return _make_request(url, varData)
+
+    if not req.ok:
+        req.raise_for_status()
+        sys.exit()
+
+    return req.json()
+
+
+def getVarConsequences(variant):
+    '''
+    Given a variant, uses Ensembl VEP API to get variant consequences
+    (e.g. intron variant, frameshift variant, missense variant)
+    using variant chromosome, Hg38 start, Hg38 end, and alternate allele as input for API
+    returns a string detailing consequences of variant
+    '''
+
+    server = "http://rest.ensembl.org"
```

Comment:
it's probably best to declare these variables outside the function if they're constant so they don't need to be recreated with every call. in this case, our standard practice is to write them as all caps e.g. `SERVER = "http://rest.ensembl.org"`